tables/tests/test_utils.py

```diff
@@ -4,7 +4,7 @@
 try:
     from unittest.mock import patch
 except ImportError:
-    from mock import patch
+    from unittest.mock import patch
 
```

Comment:
Here probably the entire try/except shall be removed, and just leave
```
from unittest.mock import patch
```

Reply:
That should be done in a different PR.